static/app/views/dashboards/sortableWidget.tsx

```diff
@@ -73,18 +74,20 @@ function SortableWidget(props: Props) {
     newlyAddedWidget,
     onNewWidgetScrollComplete,
     useTimeseriesVisualization,
+    isPrebuiltDashboard = false,
   } = props;
 
   const organization = useOrganization();
   const currentUser = useUser();
   const {teams: userTeams} = useUserTeams();
-  const hasEditAccess = checkUserHasEditAccess(
-    currentUser,
-    userTeams,
-    organization,
-    dashboardPermissions,
-    dashboardCreator
-  );
+  const hasEditAccess =
+    checkUserHasEditAccess(
+      currentUser,
+      userTeams,
+      organization,
+      dashboardPermissions,
+      dashboardCreator
+    ) && !isPrebuiltDashboard;
 
```

Comment:
Small nit, now the name `hasEditAccess` is a bit misleading imo because it now also checks against the prebuilt dashboard variable. Perhaps something like `canEdit` makes more sense